src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -2270,3 +2270,26 @@ def remove_creation_date_index(tool):
         del_index(cat, index)
 
     logger.info("Removing CreationDate index from catalogs [DONE]")
+
+
+def store_raw_snapshots(tool):
```

Comment:
This should be better named `store_raw_analyses` instead of `store_raw_snapshots`